includes/functions-extra.php

```diff
@@ -63,11 +63,20 @@ public function help_tabs() {
         get_current_screen() -> remove_help_tabs();
     }
 
-    public function howdy( $wp_admin_bar ) {
```

Comment:
No need to changes to the `howdy()` function. I am implementing a universal way to remove greeting from all languages. [See here](https://wordpress.org/support/topic/howdy-greeting/).